src/program/services/scrapers/mediafusion.py

```diff
@@ -57,39 +60,36 @@ def validate(self) -> bool:
             logger.error("Mediafusion ratelimit must be a valid boolean.")
             return False
 
-        if self.app_settings.downloaders.real_debrid.enabled:
-            self.api_key = self.app_settings.downloaders.real_debrid.api_key
-            self.downloader = "realdebrid"
-        elif self.app_settings.downloaders.torbox.enabled:
-            self.api_key = self.app_settings.downloaders.torbox.api_key
-            self.downloader = "torbox"
-        elif self.app_settings.downloaders.all_debrid.enabled:
-            self.api_key = self.app_settings.downloaders.all_debrid.api_key
-            self.downloader = "alldebrid"
-        else:
-            logger.error("No downloader enabled, please enable at least one.")
-            return False
-
         payload = {
-            "sp": {
-                "sv": self.downloader,
-                "tk": self.api_key,
-                "ewc": False
-            },
-            "sr": ["4k", "2160p", "1440p", "1080p", "720p", "480p", None],
-            "ec": False,
-            "eim": False,
-            "sftn": True,
-            "tsp": ["cached"],  # sort order, but this doesnt matter as we sort later
-            "nf": ["Disable"],  # nudity filter
-            "cf": ["Disable"]   # certification filter
+            "selected_resolutions": [
+                "4k",
+                "2160p",
+                "1440p",
+                "1080p",
+                "720p",
+                "480p",
+                None,
+            ],
+            "max_streams_per_resolution": 100,
+            "live_search_streams": True,
```

Comment:
@mhdzumair what is live search streams?

Reply:
If you enable this, it will fetch the streams from mediafusion scraper with time limit. Also note that, it's not going to fetch from scrapers every time. If you disable this, mediafusion will only respond with existing db data.